client/src/app/components/TargetCard.tsx

```diff
@@ -105,23 +108,23 @@ export const TargetCard: React.FC<TargetCardProps> = ({
     }
   };
 
-  const getImage = (): React.ComponentType => {
-    let result: React.ComponentType<any> = CubesIcon;
-    const imagePath = target?.image?.id
-      ? `/hub/files/${target?.image.id}`
-      : DefaultRulesetIcon;
-    if (target.image) {
-      result = () => (
-        <img
-          src={imagePath}
-          alt="Card logo"
-          style={{ height: 80, pointerEvents: "none" }}
-        />
-      );
+  const fetchImageAndSetDataUrl = async (imagePath: string) => {
+    try {
+      const imageDataResponse = await axios.get(imagePath, {
+        headers: {
+          Accept: "application/octet-stream",
+        },
+      });
+      const encodedSvg = encodeURIComponent(imageDataResponse.data);
```

Comment:
This will garble regular image files.  Maybe something like this: https://stackoverflow.com/a/44058739